
# Project: Smell Loss as Biologically Informed Indicator of Parkinson’s Disease

This repository contains the codes used in our study. The project applies machine learning for non-motor symptom prioritization, and utilizes transcriptomic and DNA methylation analyses to uncover molecular links with olfactory dysfunction. 
Scripts here include feature selection, classification modeling, SHAP interpretation, and WGCNA for gene module discovery. The aim is to provide reproducible workflows that connect clinical data with underlying biology in Parkinson’s disease.

## FAQ

#### What programming languages have been used?

R and Python programming language have been used in this study.

#### What IDEs have been used?

R-Studio for R and Jupyter for Python are being used.

#### What types of data were analyzed?

Clinical data (non-motor symptoms), transcriptomic (RNA-seq), and DNA methylation datasets from the PPMI database.

#### Which machine learning methods were applied?

Feature selection methods (Decision Tree, Random Forest, Boruta, NMI), Random Forest classifier, and SHAP values for feature interpretation.


## Authors

- [Md Zainul Ali]([https://www.github.com/iam-zain](https://scholar.google.com/citations?user=AMzPqiYAAAAJ&hl=en))
- [Dr Manjari Kiran]([https://www.github.com/](https://scholar.google.com/citations?user=KBmsOl8AAAAJ&hl=fr))
- [Dr Pankaj Singh Dholaniya]([https://www.github.com/](https://scholar.google.com/citations?user=h_tFfq8AAAAJ&hl=en))

## Query

For any query, contact us via  [![LinkedIn](https://img.shields.io/badge/-LinkedIn-blue)](https://www.linkedin.com/in/mdzainulali)  or  [![Gmail](https://img.shields.io/badge/-Gmail-red)](mailto:zain.crj@gmail.com).
